### DISCUSSION TITLE
C# character hexadecimal escape sequence

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#120] [Q&A] [answered]
I found on Getting Started that Objeck supports C# character unicode escape sequence (`\u`). C# also has character hexadecimal escape sequence (`\x`). Should you add support for the `\x` escape sequence also?

**Comment by objeck** [comment, marked as answer]
No plans to support this. Hex literals are supported via the '0x...'  or '0X...' syntax.
